### PR TITLE
[document]Modify the inconsistencies of grammar and wording in README_zh_CN.md

### LINKED ISSUE #14470: [Doc] Modify some incorrect sentences and spelling errors in README_zh_CN.md documents
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

image
image

### Documentation Links

https://github.com/apache/dolphinscheduler/blob/dev/README_zh_CN.md

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ahuljh**
## Purpose of the pull request

This pull request  modify mistakes of README_zh_CN.md
close #14470 
## Brief change log

Correct spelling and Syntax error

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: README_zh_CN.md [hunk 1]
```diff
@@ -54,8 +54,8 @@ DolphinScheduler 的主要特性如下：
 
 ## 参与贡献
 
-社区欢迎大家贡献，请参考此页面了解更多：[如何贡献](docs/docs/zh/contribute/join/contribute.md)，在[这里](https://github.com/apache/dolphinscheduler/contribute)可以找到good first issue
-如果你是首次贡献 dolphinscheduler。
+社区欢迎大家贡献，请参考此页面了解更多：[如何贡献](docs/docs/zh/contribute/join/contribute.md)，如果你是首次贡献 DolphinScheduler，在[这里](https://github.com/apache/dolphinscheduler/contribute)可以找到good first issue
+。
```
Comment (rickchengx) [inline README_zh_CN.md]: This period seems useless here

**Review by davidzollo — APPROVED** [review]
+1
A good beginning

**Comment by rickchengx** [comment]
rerun failed CI

**Comment by ahuljh** [comment]
@zhongjiajie ,could you please help rerun "SonarCloud Code Analysis" or check fail reason and how to resolve?

**Comment by davidzollo** [comment]
> SonarCloud Quality Gate failed

Your PR is no related with code, so there is no need to check the code quality gate

**Pull request merged**